textbook/2_grouping.qmd

```diff
@@ -0,0 +1,328 @@
+    y=alt.Y('percentage_heavy_drinkers'),
+    tooltip=['country', 'average_life_expectancy', 'percentage_heavy_drinkers']
+)
+```
+
+*Try hovering over a data point to see its country.*
+
+Why is the scatterplot misleading? Suggest one change that could provide a more nuanced picture of the relationship between alcohol consumption and life expectancy.
+
+
+:::: {.callout-default .ex-hint}
+
+Alcohol consumption is influenced by cultural norms. How could we color the data points to show this?
+
+::: {.callout-default .ex-solution}
+
+We can color each country by its geographical region:  
+
+```{python}
+
+alt.Chart(life_expectancy_df).mark_circle().encode(
+    color='region',
+    x=alt.X('average_life_expectancy', scale=alt.Scale(zero=False)),
+    y=alt.Y('percentage_heavy_drinkers'),
+    tooltip=['country', 'average_life_expectancy', 'percentage_heavy_drinkers']
+)
+```
+
+
+
+Notice that the data points form clusters by continent?
+
+While countries in both <span style="color: #E37F7E; font-weight: bold">Asia</span> and <span style="color: #9CCCC8; font-weight: bold">Europe</span> tend to have high life expectancies, heavy drinking is more common in <span style="color: #9CCCC8; font-weight: bold">Europe</span>. This makes sense - European countries often have more pronounced drinking cultures. 
+
+Meanwhile, most of the countries with low life expectancies are in <span style="color: #5D84A8; font-weight: bold">Africa</span>. Regional public health problems such as high rates of infant mortality can reduce life expectancy.
+
+By grouping the data, we have revealed that world region influences both life expectancy and alcohol consumption. This is sobering news for Viridia.
+
+:::
+
+::::
+
+:::::
+
+Coloring data points is a powerful example of a technique known as **grouping**. The goal of grouping is to distinguish between groups by basing the visual appearance of data points on the value of categorical variable. 
+
+As we have seen with alcohol consumption and life expectancy, grouping has major benefits. It allows us to:
+
+1. **Directly compare** trends and values for distinct groups
+
+2. **Discover trends** in the data that were previously obscured
+
+3. **Avoid making inaccurate conclusions** about the relationships in our data
+
+To understand these benefits, let's consider Simpson's Paradox. The paradox was not invented by Homer Simpson et al., but rather Edward H. Simpson, a British codebreaker. 
+
+Simpson reported a curious phenomenon where a trend present in several different groups of data disappears when they are combined. 
+
+Below is an illustration of the Simpson's Paradox at play. Notice how the overall dataset does not have a clear direction but when you color by group, both groups exhibit the same negative trend. 
+
+::: {.panel-tabset}
+
+### Overall
+
+```{python}
+simpsons_df = pd.read_csv("data/simpsons_paradox_data_l.csv")
+alt.Chart(simpsons_df).mark_circle().encode(
+  x=alt.X('X'),
+  y=alt.Y('Y'),
+)
+```
```

Comment:
Do we need to switch to a toy dataset here? Could we continue with the same data set, a smaller version of that data, or a related data set?